saleor/plugins/user_email/plugin.py

```diff
@@ -344,14 +344,16 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         configuration = {item["name"]: item["value"] for item in self.configuration}
         self.config = EmailConfig(
-            host=configuration["host"],
-            port=configuration["port"],
-            username=configuration["username"],
-            password=configuration["password"],
+            host=configuration["host"] or settings.EMAIL_HOST,
```

Comment:
I don't think that this is the expected way of handling this. 
This is a plugin that should be configured from the dashboard. Only the Admin plugin needs to have a configuration from settings.